plugins/dubbo/src/main/java/com/navercorp/pinpoint/plugin/dubbo/interceptor/DubboConsumerInterceptor.java

```diff
@@ -94,7 +94,7 @@ private Trace getTrace(Object target) {
             return null;
         }
 
-        return traceContext.currentTraceObject();
+        return traceContext.currentRawTraceObject();
```

Comment:
I think this is fine for the `before()` method, but `after()` method should still use `traceContext.currentTraceObject()` or have a `trace.canSampled()` check inside it as well - like in #3003 ?

Reply:
agree